Formula/clang-format.rb

```diff
@@ -44,20 +34,28 @@ class ClangFormat < Formula
   uses_from_macos "zlib"
 
   def install
-    (buildpath/"projects/libcxx").install resource("libcxx")
-    (buildpath/"tools/clang").install resource("clang")
+    if build.head?
+      ln_s buildpath/"libcxx", buildpath/"llvm/projects/libcxx"
+      ln_s buildpath/"clang", buildpath/"llvm/tools/clang"
+    else
+      (buildpath/"projects/libcxx").install resource("libcxx")
+      (buildpath/"tools/clang").install resource("clang")
+    end
 
-    mkdir "build" do
+    llvmpath = build.head? ? buildpath/"llvm" : buildpath
+
+    mkdir llvmpath/"build" do
       args = std_cmake_args
       args << "-DCMAKE_OSX_SYSROOT=/" unless MacOS::Xcode.installed?
```

Comment:
I think this line should be removed. It's handled in `std_cmake_args` - in a better fashion.

Reply:
Are you saying to ditch `llvmpath` or to not make a CMake subdirectory (or both)? Note that `llvmpath` is also used in lines 56 and onwards. I had to set `llvmpath` because `HEAD` build places `llvm` in `buildpath/"llvm"` while regular build places `llvm` in `buildpath`.